packages/fx-core/src/component/driver/file/interface/createOrUpdateDebugProfileArgs.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export interface CreateOrUpdateDebugProfileArgs {
+  target: string | undefined; // The relative path of the laucnSettings.json file
```

Comment:
When is `target` needed? Seems not in yaml schema.

Reply:
'target' is removed from createOrUpdateDebugProfileArgs.